templates/Dockerfile.tt

```
@@ -1,10 +1,11 @@
 ARG RUBY_VERSION=<%= RUBY_VERSION %>
-FROM ruby:$RUBY_VERSION-slim AS base
+FROM ruby:$RUBY_VERSION-slim-trixie@sha256:0d2adfa1930d67ee79e5d16c3610f4fbed43c98e98dbda14c2811b8197211c74 AS base
```

What was the decision against using the image without the release name?
```suggestion
FROM ruby:$RUBY_VERSION-slim@sha256:0d2adfa1930d67ee79e5d16c3610f4fbed43c98e98dbda14c2811b8197211c74 AS base
```

I know that we're subscribed to Debian releases now so we'll know when to update the image, but I think even this suggestion works better because you're always on the same release and version, and Dependabot will notify you when there's a new release.

---

I'm OK with both approaches. This is more conservative (stable) approach and less likely for contributor to unintentionally break the production. Official [docs](https://hub.docker.com/_/ruby#image-variants) also suggest using this approach
> If your image needs to install any additional packages beyond what comes with the image, you'll likely want to specify one of these explicitly to minimize breakage when there are new releases of Debian.

In any case, this decision is easy to convert at any point.

---

Good point, I agree with keeping the release name in the tag then 👍